frb_codegen/src/library/commands/flutter.rs

```diff
@@ -20,3 +21,9 @@ pub fn flutter_pub_add(items: &[String]) -> anyhow::Result<()> {
         *items
     )?)
 }
+
+#[allow(clippy::vec_init_then_push)]
+pub fn flutter_pub_get(path: &PathBuf) -> anyhow::Result<()> {
+    info!("Execute `flutter pub get` inside {path:?}");
+    check_exit_code(&command_run!(call_shell[None, None], "flutter", "pub", "get", path)?)
```

Comment:
I tried the command, it says:

```
flutter pub get rust_builder/cargokit/build_tool 
  Using a naked argument for directory is deprecated and will stop working in a future Flutter release.

  Use --directory instead.
```

What about using `--directory`

Reply:
@fzyzcjy I will fix it ASAP, I'm having some problems to execute ./frb_internal scripts, like precommit --mode slow

I don't know if it is related to my computer, I'm on a M1, but I'm investigating. The problem is related to ffigen, I don't know if you know how to solve, but sharing here.

<img width="1415" alt="Captura de Tela 2024-01-05 às 22 16 59" src="https://github.com/fzyzcjy/flutter_rust_bridge/assets/13079483/b7554ab1-c59c-4fbd-a144-2d32783c6b3c">

Reply:
@MateusHBR Firstly try to ignore it, as long as the script completes. There do exist some warnings on intel mac as well, which is unrelated to frb iirc.

Reply:
@fzyzcjy Reading the macros I noticed that the execute could open a shell and execute directly inside a specific directory, But I am having problems to execute this script, adding some breaking points, the step of:

fn modify_permissions(dart_root: &Path) -> Result<()> {
    let dir_cargokit = dart_root.join("rust_builder").join("cargokit");

    #[cfg(not(windows))]
    {
        set_permission_executable(&dir_cargokit.join("build_pod.sh"))?;
        set_permission_executable(&dir_cargokit.join("run_build_tool.sh"))?;
        set_permission_executable(&dir_cargokit.join("run_build_tool.cmd"))?;
    }

    Ok(())
}

Is not founding the directory, do you know if I should add some additional step?

'No such file or directory (os error 2)'

Reply:
I am not sure what is the exact question... But you can mimic other usage code.